pom.xml

```diff
@@ -133,6 +149,12 @@
                 <artifactId>commons-lang3</artifactId>
                 <version>3.9</version>
             </dependency>
+            <dependency>
+                <groupId>org.slf4j</groupId>
+                <artifactId>slf4j-jdk14</artifactId>
+                <version>1.7.26</version>
+                <scope>provided</scope>
```

Comment:
@dwnusbaum I this an okay way to do this?

Reply:
It's fine in that it silences the enforcer errors and should not result in the dependency being bundled unnecessarily, but the better approach is to update to 4.x of the parent POM (and update the core version to one in [this list](https://repo.jenkins-ci.org/releases/org/jenkins-ci/main/jenkins-bom/)) so that core dependencies come from a BOM associated with the Jenkins version under test so that you will never run into bogus enforcer errors caused by conflicts between versions of Jenkins dependencies as defined in the parent POM vs Jenkins itself.

When this plugin runs against old core versions in a real Jenkins instance, 1.7.25 is the version that will actually be used at runtime, so depending on 1.7.26 in `provided` scope could be dangerous if you accidentally rely on features/behavior specific to 1.7.26, since because of the issues described in [JENKINS-41827](https://issues.jenkins-ci.org/browse/JENKINS-41827), you would not notice any issues when running the test harness. That probably doesn't matter for `slf4j`, but it could matter for other dependencies.